src/data/indexData.ts

```diff
@@ -20,70 +20,81 @@ export const ProductLinkData = {
     title: 'Grow as your app evolves',
     icon: 'triangle',
     description:
-      'Expand your application capabilities with global database caching, connection pooling and real-time database events.',
+      'Expand your application capabilities with global database caching, connection pooling, and real-time database subscriptions.',
     links: [
       {
-        url: '/accelerate',
+        url: '/accelerate/what-is-accelerate',
         title: 'Prisma Accelerate',
       },
       {
-        url: '/pulse',
+        url: '/pulse/what-is-pulse',
         title: 'Prisma Pulse',
       },
     ],
   },
 }
 
 export const ORMCardLinkData = {
-  components: [
-    {
-      url: '/orm/prisma-schema',
-      title: 'Prisma Schema',
-    },
-    {
-      url: '/orm/tools/prisma-cli',
-      title: 'Prisma CLI',
-    },
-    {
-      url: '/orm/prisma-client',
-      title: 'Prisma Client',
-    },
-    {
-      url: '/orm/tools/prisma-studio',
-      title: 'Prisma Studio',
-    },
-    {
-      url: '/orm/prisma-migrate',
-      title: 'Prisma Migrate',
-    },
-  ],
-  reference: [
-    {
-      url: '/orm/reference/prisma-client-reference',
-      title: 'Prisma Client API reference',
-    },
-    {
-      url: '/orm/reference/prisma-schema-reference',
-      title: 'Prisma schema reference',
-    },
-    {
-      url: '/orm/reference/error-reference',
-      title: 'Error message reference',
-    },
-    {
-      url: '/orm/reference/prisma-cli-reference',
-      title: 'Prisma CLI reference',
-    },
-  ],
+  components: {
+    title: 'ORM Components',
+    description: 'Learn about the building blocks of Prisma ORM.',
+    links: [
+      {
+        url: '/orm/prisma-schema',
+        title: 'Prisma Schema',
+      },
+      {
+        url: '/orm/tools/prisma-cli',
+        title: 'Prisma CLI',
+      },
+      {
+        url: '/orm/prisma-client',
+        title: 'Prisma Client',
+      },
+      {
+        url: '/orm/tools/prisma-studio',
+        title: 'Prisma Studio',
+      },
+      {
+        url: '/orm/prisma-migrate',
+        title: 'Prisma Migrate',
+      },
+    ],
+  }, 
+
+  reference: {
+    title: 'Reference', 
+    description: 'Understand the capabilities and usage of our products.',
```

Comment:
```suggestion
    description: 'Understand the usage and API surface of our products.',
```

I'd consider "capabilities" to not be part of reference docs, which is much more the "dictionary" of our products (error codes, nitty gritty API stuff, etc), vs. more conceptual "capabilities". I think you might've meant "what's possible", and I think API surface covers that in this context.